packages/slice-machine/components/Simulator/components/FullPage/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { Flex } from "theme-ui";
+
+const FullPage: React.FC = ({ children }) => (
+  <Flex
+    sx={{
+      bg: "grey01",
+      top: "128px",
+      position: "absolute",
+      height: "calc(100vh - 128px)",
```

Comment:
❓ Is 128px because of the header? Could the same not be accomplished only with flex to avoid the hardcoded values?

Reply:
I'm not sure because position is absolute here

Reply:
Does it need to be absolute?